packages/console/src/components/AuditLogTable/index.tsx

```diff
@@ -40,11 +40,14 @@ function AuditLogTable({ applicationId, userId, className }: Props) {
     useSearchParametersWatcher({
       page: 1,
       event: '',
-      ...conditional(applicationId && { applicationId: '' }),
+      // If `applicationId` not specified when init this component, then search parameter of `applicationId` can be accepted.
+      ...conditional(!applicationId && { applicationId: '' }),
```

Comment:
How about fallback to `''` earlier?
```
function AuditLogTable({ applicationId = '', userId, className }: Props) {
  ...
}
```

Reply:
the input params `applicationId` and search params `applicationId` works in different cases:
input params `applicationId` is used to specify the concerned logs only related to the app, more like a fix attribute;
and search params `applicationId` can change, from one app to another, which is like a configurable filter.